### PR TITLE
feat(config.loadKubeconfig): helper for loading an API config from ~/.kube/config

### PR DESCRIPTION
**Pull request by silasbw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,13 @@ const K8Api = require('kubernetes-client');
 const k8 = new K8Api.Core(K8Api.config.getInCluster());
 ```
 
+and a helper to get the current-context config from `~/.kube/config`:
+
+```js
+const K8Api = require('kubernetes-client');
+const k8 = new K8Api.Core(K8Api.config.loadKubeconfig());
```
Comment (jcrugzz) [inline README.md]: Should this be fromKubeConfig so it's the right data structure?

Comment (silasbw) [inline README.md]: Oops, yeah, fixed.

File: lib/config.js [hunk 2]
```diff
@@ -2,6 +2,8 @@
 /* eslint no-process-env: 0 no-sync:0 */
 
 const fs = require('fs');
+const path = require('path');
+const yaml = require('js-yaml');
```
Comment (jcrugzz) [inline lib/config.js]: this now needs to be a `dependency` rather than a `devDependency`

Comment (silasbw) [inline lib/config.js]: Good catch, fixed.

**Pull request merged**